### ISSUE TITLE
Is there a backpropagation-method for all math-layers i.e. tf.fft2d?

### ISSUE THREAD

**Issue by beniroquai** [tensorflow/tensorflow#5831]
When building a deep NN, where one of the layer is represented by a fouriertransform (i.e. tf.fft), does tensorflow provide a backpropagation method for it? In this case it could/would be complex I guess. 
How does the optimizer work on that? Are there any documents which could explain this behavior? 

In theory, I understand how the gradient of a multiplication is carried out, but doing this for a more complicated function makes things more difficult for me. 

Thank you very much :)

**Comment by prb12** [comment]
From https://www.tensorflow.org/versions/r0.11/resources/index.html
> For help and support, technical or algorithmic questions, please submit your questions to Stack Overflow: https://stackoverflow.com/questions/tagged/tensorflow. You may also find answers in our FAQ, our glossary, or in the shapes, sizes and types guide. Please do not use the mailing list or issue tracker for support.
